tests/pylint/test_enforce_type_hints.py

```diff
@@ -1167,11 +1167,11 @@ class ToggleEntity(Entity):
     class _BaseVacuum(Entity):
         pass
 
-    class VacuumEntity(_BaseVacuum, ToggleEntity):
+    class StateVacuumEntity(_BaseVacuum, ToggleEntity):
```

Comment:
I don't think this is correct. `StateVacuumEntity` doesn't inherit `ToggleEntity`.